eduNEXT/eox-nelp
# feat: add notifications models and its admin file

## Description

This address this [issue](https://edunext.atlassian.net/browse/FUTUREX-394)

## Testing instructions
1. run ./manage.py lms migrate eox_nelp
2. Go to admin `admin/eox_nelp/upcomingcourseduedate/`
3. Create multiple records with different values
4. Test search funtionality
5. Test filters
6. Select some records a select the action notify selected notifications
7. Check logs severs

## Checklist for Merge
